example/src/App.tsx

```diff
@@ -51,6 +51,14 @@ const App = () => {
   const [isNativeUIMRecShowing, setIsNativeUIMRecShowing] = useState(false);
   const [statusText, setStatusText] = useState('Initializing SDK...');
 
+  // Enable the consent flow programmatically, also add
+  // NSUserTrackingUsageDescription to Info.plist
+  AppLovinMAX.setConsentFlowEnabled(true);
+  // url of private policy - mandatory
+  AppLovinMAX.setPrivacyPolicyUrl('https://your_company_name.com/privacy/');
+  // optional url of terms of service
+  AppLovinMAX.setTermsOfServiceUrl('https://your_company_name.com/terms/');
```

Comment:
I think it's fine to be more concise:

```suggestion
  // Enable the iOS consent flow programmatically - NSUserTrackingUsageDescription must be added to the Info.plist
  AppLovinMAX.setConsentFlowEnabled(true);
  AppLovinMAX.setPrivacyPolicyUrl('https://your_company_name.com/privacy/'); // mandatory
  AppLovinMAX.setTermsOfServiceUrl('https://your_company_name.com/terms/'); // optional
```